### PR TITLE
1.fixtures

### LINKED ISSUE #2: autocreate superuser on the first run
Use fixtures to auto create superuser and other model objects.

### PR DESCRIPTION
**Pull request by ZeroCoolHacker**
User can now load pre populated data into database
Fixes #2 
currently it is just User model.
username : admin@oriefy.com
password : oriefy123

### CODE REVIEW

**Comment by ShahzaibDilawar** [comment]
Its Good.

File: README.md [hunk 1]
```diff
@@ -25,16 +25,18 @@ And finally you make migrations to the database, create a super user, and run th
 ```bash
 > python manage.py makemigrations
 > python manage.py migrate
-> python manage.py createsuperuser
+> python manage.py loaddata accounts/fixtures/initial.json
 > python manage.py runserver
 ```
 
-Then visit `http://localhost:8000` to view the app. Alternatively you
```
Comment (ShahzaibDilawar) [inline README.md]: why did you delete these?

Comment (ZeroCoolHacker) [reply README.md]: we do not work on foreman right now.
